lib/improver/nbhood/vicinity.py

```diff
@@ -108,6 +109,8 @@ def __repr__(self):
             self.distance, self.neighbourhood_method, self.radii,
             self.lead_times, self.weighted_mode, self.ens_factor)
 
+    @ManageWarnings(
+        ignored_messages=["Collapsing a non-contiguous coordinate."])
```

Comment:
Do we always want to ignore this warning (filter it generally)? If so we could always also have something under lib/improver/__init__.py

Reply:
I've only put this in when I've checked where it is being raised and I know we are happy to suppress it. This is mostly when the cube being collapsed is not the cube output from the plugin, so it does not matter if the cube's metadata gets a bit messed up. In general this warning is suggesting we have created some messy metadata on the cube, so I think we shouldn't always ignore it without fixing the metadata.